src/Network/Receive/kRO/RagexeRE_2012_06_18a.pm

```diff
@@ -19,79 +19,7 @@ use base qw(Network::Receive::kRO::RagexeRE_2012_05_15a);
 
 sub new {
 	my ($class) = @_;
-	my $self = $class->SUPER::new(@_);
-	my %packets = (
-		'0983' => ['actor_status_active', 'v a4 C V5', [qw(type ID flag tick unknown1 unknown2 unknown3 unknown4)]], # 29

```

Comment:
Why?

Reply:
Is already implemented in sakexe_0 and with correct args